src/terminal/cursor/position/stack.lua

```diff
@@ -19,11 +19,11 @@ local _positionstack = {}
 -- @tparam number column
 -- @treturn string ansi sequence to write to the terminal
 -- @within Sequences
-function M.push_seq(row, column)
+function M.push_seq()
   local r, c = pos.get()
   -- ignore the error, since we need to keep the stack in sync for pop/push operations
   _positionstack[#_positionstack + 1] = { r, c }
-  return pos.set_seq(row, column)
+  return pos.set_seq(r, c)
```

Comment:
This is not correct. This now collects the current position, stores it on the stack, and then moves to that same position (where it already is located).

How about we keep the previous behaviour, but make the parameters optional.